Header header
uint8 num
detection/LidarRaw[15] data